test/error-handler.test.js

```diff
@@ -0,0 +1,43 @@
+var loopback = require('../');
+var app;
+var MockReq = require('mock-req');
+var MockRes = require('mock-res');
+var assert = require('assert');
```

Comment:
Please don't mock req/res objects, express maintainers are discouraging that. Setup a full loopback app and make a real request using supertest instead.

```js
var app = loopback();
app.use(loopback.urlNotFound());
app.use(loopback.errorHandler(/* opts */));

request(app).get('/url-does-not-exist')
  // verify that the response does or does not contain stack
```